python/py-graveyard/Portfile

```diff
@@ -70,6 +70,7 @@ py-backports-ssl_match_hostname \
                         3.5.0.1_1   33
 py-backports_abc        0.5_1       33
 py-backports.weakref    1.0.post1   34
+py-barnaba              0.1.6       34 35
```

Comment:
My understanding is that this should probably be 0.1.5_1, but perhaps @mojca can enlighten me?

Reply:
Using 0.1.6 is correct. When the version increases, you just need to add the new version. Only when the version does not increase, which is normally the case, does the revision need to be increased. As long as the "new version" logic gets triggered, that's what matters for graveyard ports.

Reply:
@mf2k Thanks for the explanation.